testbed/datasenders/stanza.go

```diff
@@ -120,7 +132,9 @@ func (f *FileLogWriter) GenConfigYAMLStr() string {
           layout: '%%Y-%%m-%%d'
         severity:
           parse_from: attributes.sev
-`, f.file.Name())
+    %s
+    %s
+`, f.file.Name(), f.retry, f.storage)
```

Comment:
I just checked what we do in other writers. It seems that we already do sth similar in the k8s writer but with a wrapper constructor approach: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2ac3b6c6ae481ad6bf4f54db76d5a9ec1e25e04c/testbed/datasenders/k8s.go#L154

Could we follow sth similar here for consistency instead of introducing the option methods?

Reply:
@ChrsMark I think the k8s writes are a "special" case where the configs changes completeley . 
When it comes to retry and storage options, I tend to think they are "extensions" to default configs and for that, options sound better. It gives us ability to test `retry_after_failure` and/or `storage `combinations with minimal changes. 

Options are also consistent with data receivers, for eg here
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/28ede1db2f69b0bd4cf6c4ee4ec999dd62b007d2/testbed/testbed/receivers.go#L88-L111

WDYT?

Reply:
FWIW, options open a ton of possibilities. We can use k8s writers with/without options with minimal changes in configs.